During setup, we should run `nexus build` so the user doesn't have to

Without this, we get errors like the following:

```sh
 FAIL  tests/requests/user/users.test.ts
  ● Test suite failed to run

    graphql/user.ts:13:7 - error TS2339: Property 'model' does not exist on type 'ObjectDefinitionBlock<"User">'.

    13     t.model.id();
             ~~~~~
```

--- a/index.js
+++ b/index.js
@@ -132,6 +132,14 @@ module.exports = (name) => {
       },
     },
     {
+      title: "Generate Nexus types",
+      task: async () => {
+        return execa("yarn", ["nexus", "build", "--no-bundle"], {
+          cwd: pkgName,
+        });
+      },
+    },
+    {
       title: "Cleanup",
       task: async () => {
         return Promise.all([

